packages/features/ee/common/server/checkLicense.ts

```diff
@@ -1,12 +1,28 @@
 import cache from "memory-cache";
+import { z } from "zod";
 
 import { CONSOLE_URL } from "@calcom/lib/constants";
 
 const CACHING_TIME = 86400000; // 24 hours in milliseconds
 
+const schemaLicenseKey = z
+  .string()
+  // .uuid() exists but I'd to fix the situation where the CALCOM_LICENSE_KEY is wrapped in quotes
+  .regex(/^\"?[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}\"?$/, {
+    message: "License key must follow UUID format: 8-4-4-4-12",
```

Comment:
May be an example UUID would help here, instead of showing the count of characters.